float32 percentage
string part
---
uint8 num_classes
fed_lr_ros/Sample[] dataset
